HACKING.md

```diff
@@ -22,12 +22,30 @@ These instructions were tested with installer version https://github.com/openshi
    CVO as it owns the configmap and it will revert your changes:
 
    ```sh
+   # if using kubectl:
    kubectl delete ds cluster-version-operator -n openshift-cluster-version
+
+   # if using oc:
+   oc delete daemonset cluster-version-operator -n openshift-cluster-version --config=auth/kubeconfig
```

Comment:
Sidenote: I usually do `export KUBECONFIG=$(pwd)/auth/kubeconfig`, then you don't have to pass that switch every time.

Reply:
thanks for the tip jlebon!